README.md

```diff
@@ -255,10 +259,18 @@ if err != nil {
 }
 ```
 
+<!-- GO EXAMPLE
+}
+-->
+
 ### Presence on a channel
 
+<!-- GO EXAMPLE
+{
+-->
+
 ```go
-page, err = channel.Presence.Get(ctx, nil)
+page, err := channel.Presence.Get(ctx, nil)
 for ; err == nil && page != nil; page, err = page.Next(ctx) {
 	for _, presence := range page.PresenceMessages() {
```

Comment:
```suggestion
	for _, presenceMessage := range page.PresenceMessages() {
```

Reply:
This is outside the scope of this PR. We can review the README as a separate task.